src/encoded/types/file.py

```diff
@@ -37,6 +66,9 @@ class File(SMAHTItem, CoreFile):
 
     SHOW_UPLOAD_CREDENTIALS_STATUSES = ("in review",)
 
+    class Collection(SMAHTItem.Collection):
+        pass
+
```

Comment:
An override is needed in this type definition in order to bind a view config to this Collection and not the one in base.py, otherwise the below routes for file with custom validators throw a configurator error

Reply:
Ah, makes sense, thanks for clarifying.